src/Tests/dotnet.Tests/CommandTests/CompleteCommandTests.cs

```diff
@@ -89,31 +89,16 @@ public void GivenASlashItSuggestsTopLevelOptions()
         public void GivenNewCommandItDisplaysCompletions()
         {
             var expected = new[] {
-                "--columns",
-                "--dry-run",
-                "--force",
                 "--help",
-                "--install",
-                "--language",
-                "--list",
-                "--interactive",
-                "--name",
-                "--nuget-source",
-                "--output",
-                "--type",
-                "--uninstall",
                 "-?",
                 "-h",
-                "-i",
-                "-l",
-                "-lang",
-                "-n",
-                "-o",
-                "-u",
                 "/?",
                 "/h",
-                "--update-check",
-                "--update-apply"
+                "install",
+                "list",
+                "search",
+                "uninstall",
+                "update"
```

Comment:
@baronfel I removed templates completion here, as on CI there is no templates installed(?). 
It would be good to check if it's expected, I would assume built-in templates should be there. 
However, on my machine it works as expected - template short names are part of tab completion suggestions.